underlay/src/main/resources/config/indexer/cmssynpuf_verily.json

```diff
@@ -6,20 +6,20 @@
       "datasetId": "cms_synthetic_patient_data_omop",
       "sqlSubstitutions": {
         "omopDataset": "bigquery-public-data.cms_synthetic_patient_data_omop",
-        "staticTablesDataset": "verily-tanagra-dev.aou_static_prep"
+        "staticTablesDataset": "prj-e-dexamundsen-hd2.aou_static_prep_us"
       }
     },
     "indexData": {
-      "projectId": "verily-tanagra-dev",
-      "datasetId": "cmssynpuf_index_060724"
+      "projectId": "prj-e-dexamundsen-hd2",
```

Comment:
not updating this in the service config since verily configs are added downstream

Reply:
Is this related to the rest of the PR in some way? If not, put it in a separate PR.

Reply:
I guess maybe this was for testing, so at a minimum that should be in the description. Is the plan for anyone on the project at Verily to use this project for testing? Does it have those permissions? I'd still rather see another change that updated all of the configs separately from some dependency changes.

Reply:
This was used for testing the indexing, however committing it will not really be useful -- in the new verily flows indexing must be done in external projects for the time being (more details in the project set up doc). I used this project because this is the guidance from central platform team - create personal experimental projects. 

Other config that was changed-- it moved index data set location to broad project. 

I can remove it this if it causes confusion.